diff --git a/node_modules/react-native-sms-android/android/build.gradle b/node_modules/react-native-sms-android/android/build.gradle
index 102a592..83eb9d0 100644
--- a/node_modules/react-native-sms-android/android/build.gradle
+++ b/node_modules/react-native-sms-android/android/build.gradle
@@ -11,12 +11,12 @@ buildscript {
 apply plugin: 'com.android.library'
 
 android {
-    compileSdkVersion 23
-    buildToolsVersion "23.0.1"
+    compileSdkVersion 34
+    buildToolsVersion "34.0.0"
 
     defaultConfig {
-        minSdkVersion 16
-        targetSdkVersion 22
+        minSdkVersion 21
+        targetSdkVersion 34
         versionCode 1
         versionName "1.0"
     }
@@ -30,5 +30,5 @@ repositories {
 }
 
 dependencies {
-    compile 'com.facebook.react:react-native:0.19.+'
+    implementation 'com.facebook.react:react-native:+'
 }
diff --git a/node_modules/react-native-sms-android/android/src/main/java/com/rhaker/reactnativesmsandroid/RNSmsAndroidPackage.java b/node_modules/react-native-sms-android/android/src/main/java/com/rhaker/reactnativesmsandroid/RNSmsAndroidPackage.java
index 5f246a9..de367a1 100644
--- a/node_modules/react-native-sms-android/android/src/main/java/com/rhaker/reactnativesmsandroid/RNSmsAndroidPackage.java
+++ b/node_modules/react-native-sms-android/android/src/main/java/com/rhaker/reactnativesmsandroid/RNSmsAndroidPackage.java
@@ -15,13 +15,8 @@ public class RNSmsAndroidPackage implements ReactPackage {
 
     @Override
     public List<NativeModule> createNativeModules(ReactApplicationContext reactContext) {
-      mModuleInstance = new RNSmsAndroidModule(reactContext);
-      return Arrays.<NativeModule>asList(mModuleInstance);
-    }
-
-    @Override
-    public List<Class<? extends JavaScriptModule>> createJSModules() {
-        return Collections.emptyList();
+        mModuleInstance = new RNSmsAndroidModule(reactContext);
+        return Arrays.<NativeModule>asList(mModuleInstance);
     }
 
     @Override
